### ISSUE TITLE
Provide physical install DVDs for sale

### ISSUE THREAD

**Issue by DemiMarie** [QubesOS/qubes-issues#4985]
**The problem you're addressing (if any)**
It is necessary to verify an installation ISO before using it.  This is not trivial, as even a DVD-R can (at least in theory) be modified (at least in certain limited ways) by a malicious drive it is inserted into.

**Describe the solution you'd like**
ITL could offer hard-mastered install DVDs.  A user could then verify the signature on multiple machines without worrying that any of them could alter the medium.

**Where is the value to a user, and who might that user be?**
The users who could gain the most from this are those who need to install QubesOS, but do not want to risk compromise during the process of downloading and/or verifying the install ISO

**Describe alternatives you've considered**
None that would work as well.

**Additional context**
This could also raise funds for QubesOS development.

**Comment by marmarek** [comment]
This could work well few years ago, but since DVD drives in our target hardware platform (laptops) are less and less available, I'm not sure if it's worth it.
On the other hand, it may be good to do a research about USB sticks with physical write protection. Or SD cards with meaningful write protection (not the switch only asking the reader to present it as read-only).
cc @petertodd

**Comment by andrewdavidwong** [comment]
> This could work well few years ago, but since DVD drives in our target hardware platform (laptops) are less and less available, I'm not sure if it's worth it.
> On the other hand, it may be good to do a research about USB sticks with physical write protection. Or SD cards with meaningful write protection (not the switch only asking the reader to present it as read-only).
> cc @petertodd

@petertodd has raised interesting questions about whether DVDs are really as immutable as they're assumed to be.
 
Kanguru is a company that sells USB drives with digitally-signed firmare and physical write-protect switches. Here's an example: https://www.kanguru.com/storage-accessories/kanguru-flashtrust-secure-firmware.shtml

As I concluded in [Installation Security Considerations](https://www.qubes-os.org/doc/install-security/):
> Considering the pros and cons of each, perhaps a USB drive with non-rewritable (or at least cryptographically-signed) firmware and a physical write-protect switch might be the option.

**Comment by DemiMarie** [comment]
Do limited Windows accounts provide enough access to verify the integrity
of the install image?

**Comment by petertodd** [comment]
@andrewdavidwong Yup, Kanguru is the read-only USB manufacturer that I would recommend. Also, there appear to be companies that will make custom USB-ROM's: https://www.acutrack.com/services/usb-on-demand.html Not clear how secure they actually are - could easily be just be a firmware toggle - but that's the claim at least.

Note that true short-run DVD replication (as opposed to duplication on DVD-R's) is actually much cheaper than you might expect - $1 to $2/disk even at quantities of just 500.
